TrackedObj[] tracked_obj
bool brake_status
